diff --git a/alpine/APKBUILD.in b/alpine/APKBUILD.in
index 96037883f5..62a19aa3a2 100644
--- a/alpine/APKBUILD.in
+++ b/alpine/APKBUILD.in
@@ -7,7 +7,7 @@ pkgdesc="Ceph is a distributed object store and file system"
 pkgusers="ceph"
 pkggroups="ceph"
 url="http://ceph.com"
-arch="x86_64"
+arch="@ARCH@"
 license="LGPL-2.1 and CC-BY-SA-1.0 and GPL-2.0 and BSL-1.0 and \
 GPL-2.0-with-autoconf-exception and BSD-3-Clause and MIT"
 depends="ceph-osd ceph-mds ceph-mgr	ceph-mon"
@@ -142,6 +142,7 @@ build() {
 		-DWITH_MANPAGE=ON \
 		-DWITH_PYTHON3=OFF \
 		-DWITH_LTTNG=OFF \
+		-DWITH_RDMA=OFF \
 		-DWITH_SYSTEM_BOOST=ON \
 		-DWITH_TESTS=${_with_tests:-OFF} \
 		|| return 1
@@ -219,8 +220,7 @@ common() {
 		rbd-replay \
 		rbd-replay-many \
 		rbdmap \
-		ceph-post-file \
-		ceph-brag
+		ceph-post-file 
 	_pkg $_datadir/ceph known_hosts_drop.ceph.com id_rsa_drop.ceph.com \
 	 	id_rsa_drop.ceph.com.pub
 	_pkg $_sysconfdir/ceph rbdmap
diff --git a/make-dist b/make-dist
index db62562402..7e31164541 100755
--- a/make-dist
+++ b/make-dist
@@ -124,6 +124,7 @@ for spec in ceph.spec.in alpine/APKBUILD.in; do
     cat $spec |
         sed "s/@VERSION@/$rpm_version/g" |
         sed "s/@RPM_RELEASE@/$rpm_release/g" |
+        sed "s/@ARCH@/$arch/g" |
         sed "s/@TARBALL_BASENAME@/ceph-$version/g" > `echo $spec | sed 's/.in$//'`
 done
 ln -s . $outfile
